working/2364 - primary constructors/feature-specification.md

```diff
@@ -731,15 +846,81 @@ means that they preserve their semantics when moved into the body.*
 
 Finally, _k_ is added to _D2_, and _D_ is replaced by _D2_.
 
+### Warnings
+
+The language does not specify warnings, but the following is recommended:
+
+A warning is emitted in the case where an identifier expression is resolved
```

Comment:
I think we should just make this an error. I can't imagine it every doing anything *useful* and will certainly just confuse users.